python/lsst/afw/detection/_footprintContinued.py

```diff
@@ -0,0 +1,68 @@
+# This file is part of afw.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from lsst.afw.image import Image
+
+import numpy as np
+
+from lsst.utils import continueClass
+
+from lsst.geom import Point2I
+from ._detection import Footprint
+
+
+__all__ = []  # import this module only for its side effects
+
+
+@continueClass
+class Footprint:  # noqa: F811
+    def extractFluxFromArray(self, image: np.ndarray, xy0: Point2I) -> float:
+        """
+
+        Parameters
+        ----------
+        image:
+            The array containing the image pixels to extract
```

Comment:
You need types on all these parameters and returns, spaces before the colon, periods at the end of sentences, and dimensions for the array.

I think we also generally don't say "The X", just "X", but I don't know that we have official guidance on this.

https://developer.lsst.io/python/numpydoc.html#array-types

```suggestion
        image : `numpy.ndarray`, (N, 2)
            Image array containing the pixels to sum over.
```

Reply:
The latest version of the documenteer no longer requires us to put types on parameters if we use type hints. Which is preferred, since the type hints are checked while types in parameters more easily bitrot.